src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -845,6 +847,9 @@ private void ReportTopLevelSyntacticRudeEdits(ArrayBuilder<RudeEditDiagnostic> d
                         {
                             Contract.ThrowIfFalse(statementPart == -1);
                             oldBody.FindStatementAndPartner(oldStatementSpan, newBody, out newStatement, out statementPart);
+
+                            // We should fine a partner statement since we are analyzing method body that has not been changed.
```

Comment:
Nit:
```suggestion
                            // We should find a partner statement since we are analyzing method body that has not been changed.
```